app/src/main/java/com/architectcoders/equipocinco/ui/MoviesFragment.kt

```diff
@@ -45,7 +45,7 @@ class MoviesFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         navController = view.findNavController()
-        viewModel.model.observe(this, Observer(::updateUI))
+        viewModel.model.observe(viewLifecycleOwner, Observer(::updateUI))
```

Comment:
@gabrielpozo ¿por qué has cambiado "_this_" por "_viewLifecycleOwner_"? Duda de total desconocimiento =(
_Fragment_ ya implementa _LifecycleOwner_ y entendía que _this_ era totalmente válido.

Reply:
si pones un "this" y  hay una rotacion por ejemplo o alguna transaccion en el backstack, una nueva instancia del fragment es creada y puede ser que la referencia al objeto fragment anterior no se elimine en memoria por el GC, creando asi un posible leaking, ya que el viewModel descargara datos tanto al nuevo fragment que esta activo como el antiguo q ya no lo esta